docs/license.md

```diff
@@ -2,6 +2,7 @@
 
 **MIT License**
 
+Copyright &copy; 2018 -      The Pony Developers
```

Comment:
Add a line break after this line to separate it from the next one (since it's Markdown). Also, there's no need for the dash after "2018" right now.